task_manager/test_task.hh

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright (C) 2022-present ScyllaDB
+ */
+
+/*
+ * SPDX-License-Identifier: AGPL-3.0-or-later
+ */
+
+#pragma once
+
+#include "task_manager.hh"
+
+class test_task : public task_manager::task::impl {
```

Comment:
We typically would call it `task_task_impl` by convention
since it's derived from task::impl

Reply:
similary for e.g. repair/compaction task implementations.